crypto/dh/dh_key.c

```diff
@@ -26,6 +26,11 @@
 # define MIN_STRENGTH 80
 #endif
 
+#if defined(OPENSSL_SYS_LINUX) && !defined(FIPS_MODULE) && defined (__s390x__)
+int s390x_mod_exp(BIGNUM *r, const BIGNUM *a, const BIGNUM *p,
+                const BIGNUM *m, BN_CTX *ctx, BN_MONT_CTX *m_ctx);
```

Comment:
This declaration should be in include/crypto/bn.h

Reply:
Moved.